include/cudaq/Support/TargetConfig.h

```diff
@@ -116,7 +116,8 @@ struct BackendFeatureMap {
 };
```

Comment:
currently, this header is not being installed. it will be needed as we progress. can we go ahead and add the cmake to get it installed now?

Reply:
Is it part of the runtime then? We'll need to move it, if so.